cmd/server-mux.go

```diff
@@ -31,7 +31,8 @@ import (
 )
 
 const (
-	serverShutdownPoll = 500 * time.Millisecond
+	serverShutdownPoll     = 500 * time.Millisecond
+	serverMaxParallelConns = 9000
```

Comment:
Can you add a comment on why this is chosen, perhaps even reference the github issue here for understanding.